.github/workflows/deploy.yaml

```
@@ -91,6 +93,18 @@ jobs:
           echo "Preview URL: $DEPLOYMENT_URL"
         working-directory: ./veterans
 
+      - name: Install Vercel CLI
+        run: npm install -g vercel
+
+      - name: Debug deployment URL
+        run: echo "Deployment URL: $DEPLOYMENT_URL"
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 100: mapping values are not allowed in this context

(syntax-check)